src/d3-utils/joins.ts

```diff
@@ -153,19 +153,21 @@ export function createLookmlJoinElement(svg: any, joinData: any, diagramDict: Di
     let disabled = Math.abs(aY - fY) < 50 ? true : false
 
     let joinedDegree = diagramDict.tableData[joinedTable][0].diagramDegree
-    let degreeMembers = diagramDict.yOrderLookup[joinedDegree].length
-    let stepPro = aY < fY && degreeMembers > 2
+    let degreeMembers = diagramDict.yOrderLookup[joinedDegree].length + 1
```

Comment:
Me thinks a lot of these lets should be consts

Reply:
You right. I'm gonna do a standalone PR for this